app.rb

```diff
@@ -1,9 +1,34 @@
 require 'sinatra/base'
+require './lib/peeps'
+require 'time'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
```

Comment:
It's always a good idea to remove test code from code you submit for code review (anywhere, not just at Makers). It keeps everything nice and clean :)